### ISSUE TITLE
Landscape tessellation update caused drastic performance hit

### ISSUE THREAD

**Issue by Hypno123** [TheIsleGame/bugs#199]
## Expected Behavior

Landscape tesselation should not be causing such a drastic hit in performance.

## Current Behavior

The recent update to landscape tessellation causes a huge performance hit, going from 60 fps to as low as 10.

https://www.youtube.com/watch?v=m13Vh44Narg

## Steps to reproduce

1. Spawn on Thenyaw.
2. Wait until sunrise/sunset.
3. Stand on a hill and look towards the marsh (where performance hit seems most apparent.) 
4. Watch fps die.

## Build Information

> **Game Version:** 0.1.38.7

> **Operating System**: Windows

> **Network Role**: Client

**Comment by dinosauriac** [comment]
Already done an optimization pass on ambient particles. Not entirely convinced that's what is causing performance problems.

**Comment by Hypno123** [comment]
Have you tried removing the particles entirely to see if it increases performance? One would assume that they caused the hit, as it began when they were added and mostly affects the marsh area.

**Comment by dinosauriac** [comment]
That was the first thing I did. Left the map with nothing but the base landscape and foliage on it. Still ran terribly, mainly during sunrise/sunset transitions. I then reverted back one version at a time through the recent commits until I found a revision where performance was stable. Disabling the recent landscape tessellation update was the only thing that produced a massive performance increase, with the worst case scenario going from on average 25fps to 90 on max settings. 

Landscape tessellation is apparently still being worked on, though I've yet to see any further updates to it over the past week or so. If we're actually going to go forward with it, we'd need some sort of graphics quality option to swap which material is in use by the landscape. Simply setting the tessellation multiplier to zero does diddly in terms of lightening the load, you actually need an entirely separate shader without any of that stuff in it.
